CONTRIBUTING.md

```diff
@@ -86,10 +86,10 @@ Unil there is a release of `kaoto-ui`, the steps are relatively complicated:
   * `yarn link`
   * `yarn build:lib`
 * Open VS Code on `vscode-kaoto` local clone folder
-  * In `package.json`, remove the line with `kaoto-ui` dependency
+  * In `package.json`, remove the line with `@kaoto/kaoto-ui` dependency
   * `yarn`
-  * Add back the line in `package.json` for `kaoto-ui` dependency
-  * `yarn link kaoto-ui`
+  * Add back the line in `package.json` for `@kaoto/kaoto-ui` dependency
+  * `yarn link @kaoto/kaoto-ui`
```

Comment:
Same as above; now that @kaoto/kaoto-ui is published as an npm package we shouldn't have to do all the steps to remove the line with `kaoto-ui` as a dependency and then `yarn link` etc. We should only have to install `@kaoto-ui/kaoto-ui` and it will pull it in automatically.

Reply:
part of the PR to use the released version https://github.com/KaotoIO/vscode-kaoto/pull/68